storage/client/client.go

```diff
@@ -18,6 +18,7 @@ import (
 	oasisConfig "github.com/oasisprotocol/oasis-sdk/client-sdk/go/config"
 	sdkTypes "github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
 
+	"github.com/oasisprotocol/nexus/analyzer/runtime/evm"
```

Comment:
dang ol storage depending on analyzer

Reply:
:shrug: you could move it to `common/types.go`? Although it looks OK enough to me as-is. The true storage type is `int`, really ... But it's nice to do the automatic cast into analyzer types, because the compiler will warn us(*) if we change the db input (= analyzer) but not the handling of the db output (= storage/client).

(*) in limited cases, Go's approach to enums is a joke, call me at 1 800 VITRIOL for more pointless Go rants

Reply:
ah yeah, would make sense to move it to common, now that it shows up in both storage and api

go has a lot of progressive ideas about preventing authors from locking down their own stuff, including via enums

Reply:
moved. also it's now TokenType instead of EVMTokenType

Reply:
correction: doesn't show up in api